.drone.yml

```diff
@@ -11,6 +11,19 @@ environment:
   VERSION: latest-20220623
 
 steps:
+- name: lint_dockerfile
+  pull: if-not-exists
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+  environment:
+    DOCKER_HOST: tcp://docker:2375
+  commands:
+  # wait for docker service to be up before running docker build
+  - n=0; while [ "$n" -lt 60 ] && [ ! "$(docker stats --no-stream)" ]; do n=$(( n + 1 )); sleep 1; done
+  - docker run --rm -i hadolint/hadolint < Dockerfile
+  when:
+    event:
+    - push
```

Comment:
Should we lint on pull requests too

Reply:
I was also thinking it would be good to see this working i.e. failing the build. Could we create a new branch based of this one and push a breaking change to see it fail the build

Reply:
I'd like to check if it's necessary to run docker in docker

Reply:
I think we should lint on pull request so have added that. Also, removed the docker in docker as per your branch.

Reply:
Added a breaking change to the Dockerfile to this branch: https://github.com/UKHomeOffice/callisto-ui-nginx/tree/EAHW-2095/add-dockerfile-lint-breaking

Reply:
@joannafawl Had a quick look at the docs but couldn't find that stated anywhere. Feel free to point me in the right direction! In the meantime changed "pull" to "pull_request" which is the correct event name!